### PR TITLE
separate dayIsYearVar getter function, format years in csv download

### PR DESCRIPTION
**Pull request by shaahmad**
I created a separate "Date" vs "Year" row header for the two different cases, but it may be ok to just use "Date" all the time. Lmk, I can change that.

### CODE REVIEW

File: charts/DataTab.tsx [hunk 1]
```diff
@@ -34,18 +34,20 @@ export class DataTab extends React.Component<{
 
         const rows: string[] = []
 
-        const titleRow = ["Entity", "Code", "Year"]
+        const titleRow = chart.yearIsDayVar
+            ? ["Entity", "Code", "Date"]
+            : ["Entity", "Code", "Year"]
```
Comment (breck7) [inline charts/DataTab.tsx]: nit: avoid repetition and only change the 3rd word:

    ["Entity", "Code", chart.yearIsDayVar ? "Date" : "Year"]

**Comment by jasoncrawford** [comment]
@shaahmad I think `Date` vs. `Year` is good for the header

**Comment by shaahmad** [comment]
I need to still address charts with two different time units on the x vs y axis (COVID cases vs GDP). But this fix will address issues with charts with just one time unit (e.g. total COVID cases)

**Comment by breck7** [comment]
Nice!

> I need to still address charts with two different time units on the x vs y axis (COVID cases vs GDP). But this fix will address issues with charts with just one time unit (e.g. total COVID cases)

Great. Seems like this will help in the interim. LGTM.

Long term csvBlob looks like the type of thing that would benefit from unit tests.

Though long term perhaps we should have some sort of "DataFrame/DataTable" class that all charts use, and then toCsv methods on those.

**Comment by breck7** [comment]
In the interim while we don't have unit tests could you share a before and after showing current CSV behavior with the new behavior?

**Comment by shaahmad** [comment]
Before:
Screen Shot 2020-03-24 at 9 55 40 PM
After:
Screen Shot 2020-03-24 at 9 55 50 PM

**Comment by danielgavrilov** [comment]
Looks great @shaahmad! I will merge & deploy this, we can address the issue with the different time units separately.

Congrats on the first PR! 🙌

**Pull request merged**